python/lsst/ts/logging_and_reporting/web_app/services/jira_service.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+import json
+
+from pydantic import BaseModel
+
+
+class JiraTicket(BaseModel):
+    url: str
+    summary: str
+    updated: datetime.datetime
+    create: datetime.datetime
+    system: str
+    status: str
+    key: str
+
+
+def extract_ticket_info(tic) -> JiraTicket:
+    return {
+        "url": f"https://rubinobs.atlassian.net/browse/{tic["key"]}",
+        "summary": tic["fields"]["summary"],
+        "updated": tic["fields"]["updated"],
+        "created": tic["fields"]["created"],
+        "status": tic["fields"]["status"]["name"],
+        "key": tic["key"]
+    }
+
+
+def get_jira_tickets(
+        dayobs_start: datetime.date,
+        dayobs_end: datetime.date,
+        telescope: str
+        ) -> list[JiraTicket]:
+    print(f"Getting Jira tickets for start: {dayobs_start}, "
+          f"end: {dayobs_end} and telescope: {telescope}")
+    # TODO: replace with code to retrieve
+    # ticket information using data adaptors
```

Comment:
For TODO comments is usually helpful (and a practice across other TSSW repositories) to create a ticket and add its key/number for reference. e.g
```suggestion
    # TODO: replace with code to retrieve
    # ticket information using data adaptors
    # See DM-12345.
```